src/caret_analyze/plot/timeseries/response_time_timeseries.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 Research Institute of Systems Planning, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from collections.abc import Sequence
+
+import pandas as pd
+
+from ..metrics_base import MetricsBase
+from ...record import RecordsInterface, ResponseTime
+from ...runtime import Path
+
+
+class ResponsetimeTimeSeries(MetricsBase):
+    """Class that provides latency timeseries data."""
+
+    def __init__(
+        self,
+        target_objects: Sequence[Path],
+        case: str
+    ) -> None:
+        super().__init__(target_objects)
+        self._case = case
+
+    def to_dataframe(self, xaxis_type: str = 'system_time') -> pd.DataFrame:
```

Comment:
Please add CARET_doc.

Ref: https://github.com/tier4/CARET_analyze/blob/a9a0c424d27b1dee2323584ecb46fbfa132547c0/src/caret_analyze/plot/timeseries/frequency_timeseries.py#L37-L57

Reply:
Fixed in https://github.com/tier4/CARET_analyze/commit/f4cd2af6e1f243a8c68186f3341424cb97eae919